### ISSUE TITLE
Remove label workflow job 

### ISSUE THREAD

**Issue by jpvajda** [newrelic/newrelic-observability-packs#201]
Quick starts that are being modified are failing this job repeatedly via forked prs and in some cases for non forked prs. 

### Steps to reproduce

See: 
https://github.com/newrelic/newrelic-observability-packs/pull/197/checks?check_run_id=3510663408

```
Run gh pr edit 197 --add-label "new-pack"
GraphQL error: Resource not accessible by integration
Error: Process completed with exit code 1.
```

Or: 

https://github.com/newrelic/newrelic-observability-packs/pull/192/checks?check_run_id=3537332971

```
Run gh pr edit 192 --add-label "support-level:newrelic"
GraphQL error: Resource not accessible by integration
Error: Process completed with exit code 1.
```

### Expected 

~~Label job runs and adds the correct labels to packs when applicable.~~

## Acceptance Criteria
- [ ] Remove this workflow, labels will be handled https://github.com/newrelic/newrelic-observability-packs/issues/218
- [ ] Support labels will be handled manually for now during the review process.

**Comment by aswanson-nr** [comment]
Using [pull_request_target](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target) as the triggering event should allow access to secrets for forked PRs. 
However, this comes with some [security concerns](https://securitylab.github.com/research/github-actions-preventing-pwn-requests/) that will need to verified before we move forward.

**Comment by jpvajda** [comment]
ah its a fork PR problem... we had this exact requirements for the Docs site I thought...

**Comment by LizBaker** [comment]
we could require that all forked PRs get approval to run workflows, and then pull_request_target would be safe. but that would also mean contributors wouldn't know validation had failed etc until someone approved the workflows running

**Comment by jpvajda** [comment]
> we could require that all forked PRs get approval to run workflows, and then pull_request_target would be safe. but that would also mean contributors wouldn't know validation had failed etc until someone approved the workflows running

How would this repo differ than how we manage fork prs on the docs site?

**Comment by aswanson-nr** [comment]
Finally had some time to read through all this again. Since the labeling workflow doesn't actually need to check out the branch/fork of the PR, moving it to pull_request_target should work perfectly.
I noticed that the generate-uuid workflow will also fail for forks and that one does need to check out the current branch/fork. The article I linked from github does have an example of how to do this securely, but it will require a little bit of work to set it up.
